cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -219,6 +220,11 @@ func (s *VMTServer) Run(_ []string) error {
 	// the certificate issue of 'doesn't contain any IP SANs'.
 	// See https://github.com/kubernetes/kubernetes/issues/59372
 	kubeletClient := s.createKubeletClientOrDie(kubeConfig, !isOpenshift)
+	caClient, err := clusterclient.NewForConfig(kubeConfig)
+	if err != nil {
+		glog.Errorf("Failed to generate correct TAP config: %v", err.Error())
+		os.Exit(1)
```

Comment:
Does any k8s cluster always have the cluster API deployed?  I assume it's optional, so perhaps not a good idea to exit Kubeturbo?

Reply:
I've tested it without any cluster API present, and we didn't exit.
This is here to detect fatal errors creating the cluster API client.